setup/configs.json

```diff
@@ -1,10 +1,11 @@
 [
   {
-    "STACK_PREFIX": "",
-    "BEDROCK_ENDPOINT": "https://bedrock-runtime.{}.amazonaws.com",
+    "STACK_PREFIX": "{ENTER_PREFERRED_STACK_NAME_PREFIX}",
+    "BEDROCK_ENDPOINT": "https://bedrock-runtime.{ENTER_PREFERRED_BEDROCK_REGION}.amazonaws.com",
```

Comment:
Bedrock Region is taken from the stack. This change will break the .replace in the file `app.py`